taxi/views.py

```diff
@@ -1,45 +1,61 @@
+from django.conf.global_settings import LOGIN_URL
```

Comment:
Consider using `from django.conf import settings` and then accessing `settings.LOGIN_URL` instead of importing `LOGIN_URL` directly from `django.conf.global_settings`. This approach is more conventional and allows for easier configuration management.